### PR TITLE
Build pipeline

### PR DESCRIPTION
**Pull request by nikithauc**

### CODE REVIEW

File: .azure-pipelines/beta-build-production.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
```
Comment (zengin) [inline .azure-pipelines/beta-build-production.yml:4]: nit: remove

**Review by zengin — APPROVED** [review]

File: .azure-pipelines/beta-build-production.yml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  inputs:
+    SourceFolder: '$(System.DefaultWorkingDirectory)'
+    Contents: |
+     **/*
+     !spec/**
+     !.azure-pipelines/**
+     !scripts/**
+     !.github/**
+     !.git/**
+     !.vscode/**
```
Comment (MIchaelMainer) [inline .azure-pipelines/beta-build-production.yml:26]: Do you want to keep typings-demo.gif in the copy?

File: scripts/GetPackageVersion.ps1 [hunk 3]
```diff
@@ -0,0 +1,4 @@
+$currentVersionString = (Get-Content  $(System.DefaultWorkingDirectory)/_TypeScript Beta Production Build/drop/package.json) -join "`n" | ConvertFrom-Json | Select -ExpandProperty "version"
```
Comment (zengin) [inline scripts/GetPackageVersion.ps1:1]: Does this work with spaces in file name?

File: .azure-pipelines/beta-build-production.yml (hunk 2 above)
Comment (nikithauc) [reply .azure-pipelines/beta-build-production.yml:23]: Added this in the contents to be skipped.

File: scripts/GetPackageVersion.ps1 [hunk 4]
```diff
@@ -0,0 +1,4 @@
+$currentVersionString = (Get-Content  $(System.DefaultWorkingDirectory)/typingsBetaBuild/drop/package.json) -join "`n" | ConvertFrom-Json | Select -ExpandProperty "version"
```
Comment (zengin) [inline scripts/GetPackageVersion.ps1:1]: Does `$(System.DefaultWorkingDirectory)` work?

Variables work with inline PowerShell scripts, but I thought you need to use environment variables to access these in files.

File: .azure-pipelines/beta-build-production.yml (hunk 1 above)
Comment (nikithauc) [reply .azure-pipelines/beta-build-production.yml]: Done

**Review by MIchaelMainer — APPROVED** [review]

File: scripts/GetPackageVersion.ps1 (hunk 4 above)
Comment (nikithauc) [reply scripts/GetPackageVersion.ps1]: Changed this.

**Review by zengin — APPROVED** [review]

**Pull request merged**